packages/renderer/src/lib/preferences/PreferencesConnectionCreationRendering.svelte

```diff
@@ -45,6 +45,7 @@ export let callback: (
   tokenId?: number,
 ) => Promise<void>;
 export let taskId: number | undefined = undefined;
+export let isEmbedded = false;
```

Comment:
hi, I'm not convinced about a 'isEmbedded' flag because AFAIK components can always be included in some other components.

It's just that when including a component you want to include it in different forms.

I would prefer options to disable/enable options

like `enableEmptyScreen : false | true` (or another property name)

`displayCloseButtonAfterCreation : true | false` , etc

``

then, it's where we use the component that we can customize the data.

because one might want to hide some items when including it, another want to display it